src/components/address-input/_macro.njk

```diff
@@ -160,12 +160,12 @@
                             "errorMessageAPI": params.errorMessageAPI,
                             "errorMessageAPILinkText": params.errorMessageAPILinkText,
                             "options": params.options,
-                            "manualLink": params.manualLink,
+                            "manualLinkUrl": params.manualLinkUrl,
                             "manualLinkText": params.manualLinkText
                         })
                     }}
                     {% if params.manualLinkText %}
-                        <a href="{{ params.manualLink | default('#0') }}" class="ons-js-address-manual-btn ons-u-mt-s ons-u-dib"
```

Comment:
How come we are removing the default here?

Reply:
This was mentioned in the ticket tech notes to remove the default link and add it in the example instead.